ui/golemOperation/OperationSubHeading.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@chakra-ui/react';
+import React from 'react';
+
+import TxEntity from 'ui/shared/entities/tx/TxEntity';
+
+type Props = {
+  txHash: string;
+  opIndex: string;
+};
+
+const OperationSubHeading = ({ txHash, opIndex }: Props) => {
+  return (
+    <Box display="flex" alignItems="center" w="100%">
+      <TxEntity hash={ txHash } truncation="none"/>, index { opIndex }
```

Comment:
We should make a new component that accepts `txHash` & `opIndex` and links to the operation page.
Base it on `TxEntity`/`StorageEntity` etc.

Reply:
Once you create this component, implement it inthe  operations list.